mlcomp/setup.py

```diff
@@ -10,7 +10,8 @@
 # The text of the README file
 README = open(ROOT + "/README.md").read()
 
-install_requires = ['ml-ops', 'termcolor', 'flask', 'flask_cors', 'psutil', 'py4j', 'nbformat', 'nbconvert']
+install_requires = ['ml-ops', 'termcolor', 'flask', 'flask_cors', 'psutil', 'py4j', 'nbformat', 'nbconvert',
+                    'uwsgidecorators']
```

Comment:
now we need to update mcenter_runner and reflex-algos builder containers?

Reply:
For reflex-algos - yes, assuming we'll add unit-test for uwsgi with mlpiper, which would run during build. As for the mcenter_runner - it's already installed.